deploy/docs/build.sh

```diff
@@ -14,21 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -exu
 
-## This script builds the Skaffold site assuming it's ran from a
-## cloned Skaffold repo with no submodules initialized. The script initializes the git submodules for
-## the site's theme in a standard manner, thus this script can be used locally as well as for the PR review flow.
-set -x
+readonly BASE_URL=${1}
 
-readonly DOCSY_COMMIT=$(git config -f .gitmodules submodule.docsy.commit)
-readonly REPO_DIR=$(pwd)
-readonly BASE_URL=${1:-"http://localhost:1313"}
-
-git submodule init && \
-git submodule update --init && \
-cd  docs/themes/docsy && \
-git checkout ${DOCSY_COMMIT} && \
-git submodule update --init --recursive && \
-cd  ${REPO_DIR}/docs && \
-npm i -D autoprefixer && \
+cd docs
+mkdir themes
```

Comment:
```suggestion
mkdir -p themes
```

Reply:
currently the controller is failing on this step. Can we try with `-p`?

Reply:
![image](https://user-images.githubusercontent.com/635073/54167218-e6e8b580-4425-11e9-9c1d-3e69b7595f87.png)

Reply:
Thank you @balopat for investigating.

I don't think there's an issue with `themes` directory.
I think the problem is that it uses an *old* version of the `docs-controller` image. This PR changes the image. I don't know how to handle that case properly

Reply:
You can temporarily deploy your version of the controller using `gcloud submit deploy/webhook/cloudbuild.yaml --project k8s-skaffold` or similar

Reply:
so...the trick was that the docs-controller image was not pushed/published only with the `deploy/docs/cloudbuild.yaml` - after I did that, we can run `deploy/webhook/cloudbuild.yaml` which then deploys the new controller  but more importantly it can spin up the previews with the right base-image.

Reply:
I'm not a big fan of this setup - I should sit down and design the proper dependency graph of these steps soon...